src/common/configuration.ts

```diff
@@ -15,6 +15,7 @@ export interface Configuration {
   // package config
   ib_host: string;
   ib_port: number;
+  ib_test_account: string;
```

Comment:
This variable needs to be added to sample.env file aswell

Reply:
Done